core/domain/stats_services.py

```diff
@@ -77,7 +91,12 @@ def get_versions_for_exploration_stats(exploration_id):
 def get_exploration_stats(exploration_id, exploration_version):
     """Returns a dict with state statistics for the given exploration id.
 
-    Note that exploration_version should be a string.
+    Args:
+        exploration_id: str. The exploration ID.
+        exploration_version: str. The version of the exploration.
```

Comment:
In get_exploration_stats(exploration_id, exploration_version) function, exploration_version = '3' and exploration_version = 3 gives same result, But documenting it as a string for this function gives possibility to use exploration_version = 'all' which is also a valid argument. I used the interactive console feature to test it with inputs '3', 3 and 'all'.